.gitignore

```diff
@@ -21,35 +11,66 @@
 build/
 gen/
 
-# Gradle files
-.gradle/
-build/
+#Exclude .pem files to make sure they aren't checked in
+*.pem
```

Comment:
Definitely safe and generally true, so leave it for now.

I will be checking in the pem for our demo study at some point though. We want people to be able to check out our code and play with it without doing a bunch of setup. So BridgeAndroidSDK and BridgeResearchStackSampleApp